packages/components/icons/src/generated/AngleDownReact.tsx

```diff
@@ -1,6 +1,8 @@
 // THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
 // This file is created with 'yarn generate-icons'.
 // Original SVG file: src/svg/angle-down.react.svg
+import { useMemo } from 'react';
+import { createSequentialId } from '@commercetools-uikit/utils';
 /* eslint-disable @typescript-eslint/no-unused-vars */
 import { warning } from '@commercetools-uikit/utils';
```

Comment:
Nit: It looks like we're importing from the same package twice.

Reply:
@emmenko I moved the imports to `icons.styles.tsx` and that seems to fix it. You can see it here - [3846b48](https://github.com/commercetools/ui-kit/pull/2524/commits/3846b489475daa819e0b5075233aad38ac75cc7d).
Let me know if it all looks good now 😄